### ISSUE TITLE
netcdf-c

### ISSUE THREAD

**Issue by ngam** [conda-forge/libnetcdf-feedstock#132]
Other places (e.g. spack.io) call this netcdf-c rather than libnetcdf. Can we add an alias or something, to be fully consistent with netcdf-fortran and other providers? Thanks!

- [x] I read [the conda-forge documentation](https://conda-forge.org/docs/user/introduction.html#how-can-i-install-packages-from-conda-forge) and could not find the solution for my problem there.

Issue:

<br/>
Environment (<code>conda list</code>):
<details>

```
$ conda list

```
</details>

<br/>
Details about  <code>conda</code> and system ( <code>conda info</code> ):
<details>

```
$ conda info

```
</details>

**Comment by ocefpaf** [comment]
@ngam note that I'd love to rename but this is one of those places where a bad choice needs to be maintained b/c changing out be way more disruptive. Note that we, conda-forge, did not choose that. We inherited it from defaults and kept the same name to be compatible with the main channel.

**Comment by ngam** [comment]
Yes, I understand and I am NOT in favor of renaming (because it would be highly disruptive).

I was thinking we could have an alias. I tried looking into it and I realized a possibility would be to just output this twice (like I did here: https://github.com/conda-forge/coreutils-feedstock/pull/23) but that would also be unnecessarily redundant? Hence, closing... happy to submit a PR to split it into two outputs if you think it is worth it :)

**Comment by ngam** [comment]
Or maybe we could have a meta package ... that would be pretty easy, but again, it might set a bad precedent with people submitting meta packages like that all over the place

**Comment by ocefpaf** [comment]
I created a few aliases packages with a meta package that installs the canonical one. So, netcdf-c, would install libnetcdf-c. That is IMO, the easiest solution b/c you avoid duplicated builds. However, after making a few of those for Python packages with `-` and `_`. I kind of gave up b/c that still confused a few users.

TL;DR I'm not against it and I can help review a PR but I'm not sure the effort is worth it.
